tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -1938,12 +1978,18 @@ def _test_enable_fast_capacity_failover(
     clustermgtd_conf_path,
     static_nodes_in_ice_compute_resource,
     ice_dynamic_nodes,
+    target_compute_resource,
 ):
     # set insufficient_capacity_timeout to 180 seconds to quicker reset compute resources
     _set_insufficient_capacity_timeout(remote_command_executor, 180, clustermgtd_conf_path)
-    # trigger insufficient capacity
+
+    # clear slurm_resume and clustermgtd logs in order to start from a clean state
+    remote_command_executor.run_remote_command("sudo truncate -s 0 /var/log/parallelcluster/slurm_resume.log")
```

Comment:
NIT - what about creating an utility function out of this, instead of repeating the code multiple times?

Reply:
Makes sense.